.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+version: 2
+
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: weekly
+
+  - package-ecosystem: "maven"
+    directory: "/"
+    schedule:
+      interval: weekly
```

Comment:
wouldn't be better to group updates into one PR instead of having many PR's with version update? like there for reference https://github.com/skodjob/test-frame/blob/c5962d6ba6dc2642cbb364439f708bd58361e1f8/.github/dependabot.yml#L14

Reply:
that's a good idea!